src/main/java/edu/cornell/kfs/module/ar/document/service/impl/CuContractsGrantsInvoiceDocumentServiceImpl.java

```diff
@@ -67,6 +104,22 @@
         return templateParameters;
     }
     
+    protected String convertDateToString(Date dateValue) {
+        String dateValueAsString = StringUtils.EMPTY;
+        if (ObjectUtils.isNotNull(dateValue)) {
```

Comment:
Did you run into NPEs during testing.  The super doesn't do null checks when setting Date value, it just calls getDateTimeService().toDateString()?

Reply:
I was running into data not populating into the template when date was null. It could be the state of the data in  DEV since I used TEMP spun up from copy of DEV.  I will remove and let future validation efforts confirm whether it needs to be added back in.